http-transport-binding.md

```diff
@@ -58,26 +59,33 @@ which is compatible with HTTP 1.1 semantics.
 
 ### 1.3. Content Modes
 
-This specification defines two content modes for transferring events:
-*structured* and *binary*. Every compliant implementation SHOULD support both
-modes.
-
-In the *structured* content mode, event metadata attributes and event data are
-placed into the HTTP request or response body using an [event
-format](#14-event-formats).
+This specification defines three content modes for transferring events:
+*binary*, *structured* and *batched*. Every compliant implementation SHOULD
+support the *structured* and *binary* modes.
 
 In the *binary* content mode, the value of the event `data` attribute is placed
 into the HTTP request or response body as-is, with the `contenttype` attribute
 value declaring its media type; all other event attributes are mapped to HTTP
 headers.
 
+In the *structured* content mode, event metadata attributes and event data are
+placed into the HTTP request or response body using an [event
+format](#14-event-formats).
+
+In the *batched* content mode several events are batched into a single HTTP
+request or response body using an [event format](#14-event-formats) that
+supports batching.
+
 ### 1.4. Event Formats
 
 Event formats, used with the *structured* content mode, define how an event is
 expressed in a particular data format. All implementations of this
 specification MUST support the [JSON event format][JSON-format], but MAY
```

Comment:
do we need to make this clearer w.r.t. which json format we mean? single vs batched - or do they have to support both?

Reply:
@cneijenhuis what do you think about making this:
```
... MUST support the non-batching [JSON event format][JSON-format], but MAY...
```
so just add `non-batching`  ?

Reply:
I like it! https://github.com/cloudevents/spec/pull/370/commits/d3f2edbd4837dbc6194aa418059d820669392523

Reply:
thanks